CONTRIBUTING.md

```diff
@@ -1,19 +1,104 @@
-# Contributing
+# Contributing guidelines
 
 When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change. 
+email, or any other method with the owners of this repository before making a change.
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
+### Setting things up
+
+To set up the development environment, follow the instructions in README.
```

Comment:
it would be a great user experience to add a hyperlink to README.

Reply:
Done